src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -136,6 +137,15 @@ private void changeCheckInServicePointId(UUID servicePointId) {
     write(representation, "checkinServicePointId", servicePointId);
   }
 
+  private void changeItemStatusToDeclaredLost(){
+
+    Item item = getItem();
+    if (item != null) {
+      item.changeStatus(ItemStatus.LOST);
+    }
+    changeItemStatus(ItemStatus.LOST.getValue());
```

Comment:
I don't think it is necessary to do this here, there is a line in the loanRepository that uses the status from the item when storing the record.

I acknowledge this is confusing because the `changeItemStatus` method is still present in the loan class. It probably should be removed. 

If this is too big a change for this pull request, I understand that, however I would prefer if it was possible to avoid using this method more.